src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -814,7 +816,18 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 		const disposables = new DisposableStore();
 		let codeBlockIndex = 0;
 
-		markdown = new MarkdownString(markdown.value, {
+		// inject keybinding hints for command links
+		const value = markdown.value.replace(/\[([^\]]+)\]\(command:([^\)]+)\)/g, (match, label, command) => {
```

Comment:
I would prefer doing this on the other side of the markdown renderer, like https://github.com/microsoft/vscode/blob/5f86283e88d69f36b65a5f1c3a8b4d7531b2cafd/src/vs/workbench/contrib/chat/browser/chatMarkdownDecorationsRenderer.ts#L32, because then we could be sure that the command link was actually allowed and not stripped by the sanitizer, or that it was actually a link and not something in a codeblock. This is fine for now

Reply:
https://github.com/microsoft/vscode/pull/200191